system-tests/lib/performance-reporter.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path')
+const chalk = require('chalk')
+const Libhoney = require('libhoney')
+
+const pkg = require('@packages/root')
+const ciProvider = require('@packages/server/lib/util/ci_provider')
+const { commitInfo } = require('@cypress/commit-info')
+
+class StatsdReporter {
+  constructor (runner) {
+    if (!process.env.HONEYCOMB_API_KEY) {
+      return
+    }
+
+    console.log(chalk.green('Reporting to honeycomb'))
+    this.honey = new Libhoney({
+      dataset: 'systemtest-performance',
+      writeKey: process.env.HONEYCOMB_API_KEY,
+    })
+
+    runner.on('test', (test) => {
+      test.wallclockStart = Date.now()
+    })
+
+    runner.on('test end', (test, done) => {
+      // Skipped tests never get a 'start' event, but they still get 'test end' somehow.
+      if (test.state === 'skipped') {
+        return
+      }
+
+      commitInfo().then((commitInformation) => {
+        const ciInformation = ciProvider.commitParams() || {}
+        const [, testTitle, browser] = test.title.match(/(.+?)(?: \[([a-z]+)\])?$/)
+
+        const honeycombEvent = this.honey.newEvent()
+
+        honeycombEvent.timestamp = test.wallclockStart
+        honeycombEvent.add({
+          test: testTitle,
```

Comment:
I don't know if this is enough - I think we need the fully qualified test path:
`test.titlePath().join(' /// ')` otherwise we're missing the suite info

Reply:
Ah, that's exactly what I was looking for, thanks.